crt_portal/cts_forms/tests/test_commands.py

```diff
@@ -95,3 +96,49 @@ def test_update_repeat_writer_info(self):
         call_command('generate_repeat_writer_info')
         repeat_writer_1 = RepeatWriterInfo.objects.filter(email=self.email1.upper()).first()
         self.assertEqual(repeat_writer_1.count, 105)
+
+
+class GenerateYearlyReports(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create_user("DELETE_USER", "ringo@thebeatles.com", "")
+        # Create 100 reports that all have the same violation summary
+        self.email1 = SAMPLE_REPORT_1["contact_email"]
+        self.email2 = SAMPLE_REPORT_2["contact_email"]
+        self.email3 = SAMPLE_REPORT_3["contact_email"]
+        self.email4 = SAMPLE_REPORT_4["contact_email"]
+        for _ in range(100):
+            Report.objects.create(**SAMPLE_REPORT_1)
+        for _ in range(50):
+            Report.objects.create(**SAMPLE_REPORT_2)
+        for _ in range(50):
+            Report.objects.create(**SAMPLE_REPORT_3)
+        Report.objects.create(**SAMPLE_REPORT_4)
+        call_command('generate_yearly_reports')
+
+    def test_reports_created(self):
+        reports_count = ReportsData.objects.all().count()
+        year_range = list(range(2020, datetime.today().year + 1))
+        self.assertEqual(reports_count, len(year_range))
+
+    def test_reports_data_view(self):
```

Comment:
@billyhunt Replace this test with a test that confirms the number of reports in each yearly report with the created sample reports above.